.drone.star

```diff
@@ -1079,7 +1079,7 @@ def wopiValidatorTests(ctx, storage, wopiServerType, accounts_hash_difficulty =
                          "image": OC_CI_ALPINE,
                          "environment": {},
                          "commands": [
-                             "curl -v -X PUT 'https://ocis-server:9200/remote.php/webdav/test.wopitest' -k --fail --retry-connrefused --retry 7 --retry-all-errors -u admin:admin -D headers.txt",
```

Comment:
@individual-it @SagarGi Is `test.wopitest` name can be changed across all tests or this is a predefined file name for the wopi API validator?

Reply:
I think it's only used here, so your change should everything that needs to be changed

Reply:
The .wopitest is an extension for the validator, specially on ms 365